rollout/workloadref_utils.go

```diff
@@ -0,0 +1,23 @@
+package rollout
+
+import (
+	"github.com/argoproj/argo-rollouts/pkg/apis/rollouts/v1alpha1"
+)
+
+// isProgressiveMigrationComplete checks if the progressive scale down migration is complete,
+// returning true if the rollout is healthy and the workload reference is set to scale down progressively.
+func (c *rolloutContext) isProgressiveMigrationComplete() bool {
```

Comment:
I am not sure how this means that the progressive migration is complete, I am not sure the name of the function is bad or maybe just needs a different description of what it means and why?

Reply:
To me this function just means WorkloadLoad ref is configured with ScaleDownProgressively and the Rollout is healthy. I am not sure how this means a migration is complete?